src/main/java/com/exasol/release/robot/repository/GitHubRepositoryContentFactory.java

```diff
@@ -26,8 +27,8 @@ static synchronized GitHubRepositoryContentFactory getInstance() {
     }
 
     /**
-     * Create a new instance of {@link GitBranchContent} from a GitHub repository depending on the repository's
-     * layout. Currently always returns {@link JavaMavenGitBranchContent}.
+     * Create a new instance of {@link GitBranchContent} from a GitHub repository depending on the repository's layout.
```

Comment:
Make sure that you have enabled formating before commit or on save. It looks like your IDE missed some formating on the previous commit.